lang/golang/golang-package.mk

```diff
@@ -230,8 +230,10 @@ GO_PKG_DEFAULT_ASMFLAGS= \
 
 GO_PKG_DEFAULT_LDFLAGS= \
 	-buildid '$(SOURCE_DATE_EPOCH)' \
-	-linkmode external \
-	-extldflags '$(patsubst -z%,-Wl$(comma)-z$(comma)%,$(TARGET_LDFLAGS))'
+	$(if $(GO_PPC64),, \
```

Comment:
I'm probably misunderstanding the logic here, but why not check `CGO_ENABLED` instead in case there are other architectures that it needs to be disabled for in the future?

Reply:
but why not check `CGO_ENABLED` instead

`CGO_ENABLED` is inside `GO_PKG_TARGET_VARS` and no way to check it with such simple if statement.

> other architectures that it needs to be disabled for in the future

let's leave it to then. though it's unlikely to happen, as even "newly" added loong64 and riscv64 have CGO support.

Reply:
Thanks. It was a brainfart: I didn't look at the full code in the changes and assumed it was a separate variable :facepalm: